app/helpers/application_helper.rb

```diff
@@ -1,2 +1,3 @@
 module ApplicationHelper
+  include Pagy::Frontend
```

Comment:
why do we need this?

Reply:
It is required for page_nav function in index.html.erb file